doc/README.md

```diff
@@ -0,0 +1,7 @@
+# Seed Spring Boot Documentation
+
+## Technical
```

Comment:
I would suggest adding a link to the REST API specifications (web page generated using swagger) of the systelab-doc repository. I think that this would help to give more visibility to the "contract" shared between the different seed projects.

Reply:
Done.